pkg/storage/cassandra_dependencies.go

```diff
@@ -36,26 +36,64 @@ func cassandraDeps(jaeger *v1.Jaeger) []batchv1.Job {
 		jaeger.Spec.Storage.CassandraCreateSchema.Mode = "prod"
 	}
 
+	envVars := []corev1.EnvVar{{
+		Name:  "MODE",
+		Value: jaeger.Spec.Storage.CassandraCreateSchema.Mode,
+	}, {
+		Name:  "DATACENTER",
+		Value: jaeger.Spec.Storage.CassandraCreateSchema.Datacenter,
+	}}
+
 	host := jaeger.Spec.Storage.Options.Map()["cassandra.servers"]
 	if host == "" {
 		jaeger.Logger().Info("Cassandra hostname not specified. Using 'cassandra' for the cassandra-create-schema job.")
 		host = "cassandra" // this is the default in the image
 	}
+	envVars = append(envVars, corev1.EnvVar{
+		Name:  "CQLSH_HOST",
+		Value: host,
+	})
 
 	port := jaeger.Spec.Storage.Options.Map()["cassandra.port"]
 	if port == "" {
 		jaeger.Logger().Info("Cassandra port not specified. Using '9042' for the cassandra-create-schema job.")
 		port = "9042" // this is the default in the image
 	}
+	envVars = append(envVars, corev1.EnvVar{
+		Name:  "CQLSH_PORT",
+		Value: port,
+	})
 
 	keyspace := jaeger.Spec.Storage.Options.Map()["cassandra.keyspace"]
 	if keyspace == "" {
 		jaeger.Logger().Info("Cassandra keyspace not specified. Using 'jaeger_v1_test' for the cassandra-create-schema job.")
 		keyspace = "jaeger_v1_test" // this is default in the image
 	}
 
-	username := jaeger.Spec.Storage.Options.Map()["cassandra.username"]
-	password := jaeger.Spec.Storage.Options.Map()["cassandra.password"]
+	envVars = append(envVars, corev1.EnvVar{
+		Name:  "KEYSPACE",
+		Value: keyspace,
+	})
+
+	var envFromSource []corev1.EnvFromSource
+	if len(jaeger.Spec.Storage.SecretName) > 0 {
```

Comment:
There is a function on utils that we can reuse to create this 

https://github.com/jaegertracing/jaeger-operator/blob/master/pkg/util/util.go#L280

Reply:
Neat! Thanks for the hint, I'll replace it. And thanks for reviewing 👍